test/kwc-building-listing_test.html

```diff
@@ -10,6 +10,53 @@
     <script src="../../web-component-tester/browser.js"></script>
 
     <link rel="import" href="../kwc-building-listing.html">
+
+    <script>
+            var challenges = [{
+                "title": "Chatterbox",
+                "slug": "pixel_talking_mouth",
+                "image": "https://www.tonymacx86.com/data/avatars/l/1342/1342909.jpg"
+            },
+            {
+                "title": "Virtual fireplace",
+                "slug": "pixel_fireplace",
+                "image": "https://rfclipart.com/image/thumbnail/a8-2f-da/symbolic-stone-fireplace-Download-Royalty-free-Vector-File-EPS-32405.jpg"
+            },
+            {
+                "title": "Critters!",
+                "slug": "pixel_critter",
+                "image": "https://i.pinimg.com/736x/ab/24/e7/ab24e7ed83e6d4523f10c790bd6ffc9b--zoo-animals-wild-animals.jpg"
+            },
+            {
+                "title": "Roll the dice",
+                "slug": "pixel_dice",
+                "image": "https://merribellmusic.files.wordpress.com/2015/03/black-and-white-dice-clipart-xcgk7kpmi.png?w=150&h=150"
+            },
+            {
+                "title": "Light wander",
+                "slug": "pixel_wandering",
+                "image": "http://images.clipartpanda.com/light-bulb-clip-art-light-bulb-clip-art-1-150x150.jpg"
+            },
+            {
+                "title": "Weather station",
+                "slug": "pixel_weather",
+                "image": "https://simpleclipart.com/image/thumbnail/cb-fb-48/cloud-and-sun-weather-icon-Download-Royalty-free-Vector-File-EPS-52116.jpg"
+            },
+            {
+                "title": "Shy light",
+                "slug": "pixel_shy_light",
+                "image": "http://l.thumbs.canstockphoto.com/canstock0735539.jpg"
+            },
+            {
+                "title": "The Aurora Borealis",
+                "slug": "pixel_aurora_borealis",
+                "image": "https://rfclipart.com/image/thumbnail/f6-68-b2/arctic-glacier-landscape-against-full-moon-in-the-sky-Download-Royalty-free-Vector-File-EPS-88824.jpg"
+            }],
+            completedChallenges = [
+                "pixel_talking_mouth",
+                "pixel_dice"
+            ];
+        </script>
```

Comment:
Is this exactly the same one as the demo? Could we just chuck them both in a shared JS file and load that in?

Reply:
Just me copy and pasting. It doesn't have to be the same and I don't think a change to the demo should change the test data as a side effect.